src/handlers/AccountServices.js

```diff
@@ -59,7 +59,7 @@ class AccountServices {
       backoff: 2,
       max_tries: RETRY_INVITE_COUNT
     }).then( () => {
-      var rroom = new RemoteRoom("service_"+event.sender);
+      const rroom = new RemoteRoom("service_"+event.sender);
```

Comment:
Yes, one of the side effects of messing with multiple projects. Another rule I aught to use.